test/format/config.c

```diff
@@ -499,15 +499,21 @@ config_checksum(void)
     /* Choose a checksum mode if nothing was specified. */
     if (!config_is_perm("disk.checksum"))
         switch (mmrand(NULL, 1, 10)) {
-        case 1: /* 10% */
+        case 1:
+        case 2:
+        case 3:
+        case 4: /* 40% */
             config_single("disk.checksum=on", false);
             break;
-        case 2: /* 10% */
+        case 5: /* 10% */
             config_single("disk.checksum=off", false);
             break;
-        default: /* 80% */
+        case 6: /* 10% */
             config_single("disk.checksum=uncompressed", false);
             break;
+        default: /* 40% */
```

Comment:
10% for uncompressed and 40% for unencrypted seem skewed. Why not something more balanced?

Reply:
I think "on" and "unencrypted" are the most "likely correct" settings for applications, so I gave them each 40%. My opinion at the moment is that "uncompressed" is dangerous, I don't know of any compression engine we can trust.